### DISCUSSION TITLE
Explore doesn't function properly

### DISCUSSION THREAD

**Discussion by ghost** [community/community#126180] [Mobile]
### Select Topic Area

Bug

### Body

before I was able to see in explore my favourite and releases and now even when i enable them on the filter nothing i can see same problem with the website can you fix it plz !?!!
![Screenshot_2024-05-28-09-35-36-016_com github android](https://github.com/community/community/assets/124626266/dd64bea4-e74c-4563-baa7-da28440cdbcb)
![Screenshot_2024-05-28-09-35-51-203_com github android](https://github.com/community/community/assets/124626266/9e40cbe9-8fcc-4511-b9a6-4cd3e0a311e4)
 can see same problem with the website can you fix it plz !?!!

**Comment by taki-on** [comment]
@Wadi3002 Thank you for reporting it. It seems like an API issue. I will pass it to the corresponding team.

**Reply by ghost** [reply]
Hopefully they will fix it soon thx
